pallets/loan/src/lib.rs

```diff
@@ -33,14 +35,36 @@ mod mock;
 #[cfg(test)]
 mod tests;
 
+mod loan_type;
 pub mod math;
 
+/// The data structure for storing pool nav details
+#[derive(Encode, Decode, Copy, Clone, PartialEq, Default)]
+#[cfg_attr(feature = "std", derive(Serialize, Deserialize, Debug))]
+pub struct NAVDetails<Amount> {
+	// this is the latest nav for the given pool.
+	// this will be updated on these scenarios
+	// 1. When we are calculating pool nav
+	// 2. when there is borrow or repay or write off on a loan under this pool
+	// So NAV could be
+	//	approximate when current time != last_updated
+	//	exact when current time == last_updated
+	latest_nav: Amount,
+
+	// this is the last time when the nav was calculated for the entire pool
+	last_updated: u64,
```

Comment:
no, we will alway use epoch time

Reply:
Can we have a new type then here? Or an alias at least?

Reply:
I maybe missing something here but does new type or alias solve anything specifically ?

Reply:
No, just don't like raw types. More expressive.

Reply:
I personally prefer raw type for this epoch since it straightforward. If you look at the older versions, I used `Moment` associate type but felt cumbersome with conversions. 
Defining a type alias is fine as long as we dont place them under `runtime-common` as we dont want hard dependency on that in this pallet. Either this is not final. If we absolutely dont like it, we can pretty much change it later